cmd/tsbs_load_victoriametrics/main.go

```diff
@@ -0,0 +1,69 @@
+// tsbs_load_victoriametrics loads a VictoriaMetrics with data from stdin.
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"flag"
+	"log"
+	"strings"
+	"sync"
+
+	"github.com/timescale/tsbs/load"
+)
+
+// Global vars
+var (
+	loader  *load.BenchmarkRunner
+	bufPool sync.Pool
+	vmURLs  []string
+)
+
+// Parse args:
+func init() {
+	bufPool = sync.Pool{
+		New: func() interface{} {
+			return bytes.NewBuffer(make([]byte, 0, 16*1024*1024))
+		},
+	}
+	loader = load.GetBenchmarkRunner()
+	var urls string
+	flag.StringVar(&urls, "urls", "http://localhost:8428/write",
+		"Comma-separated list of VictoriaMetrics ingestion URLs(single-node or VMInsert)")
+	flag.Parse()
+
+	if len(urls) == 0 {
+		log.Fatalf("missing `urls` flag")
+	}
+	vmURLs = strings.Split(urls, ",")
```

Comment:
should we add validation for urls here?

Reply:
Don't think so. Wrong values will be immediately reported on attempt to create http request.